### PR TITLE
Batch Template Wizard - First Changes

### PR DESCRIPTION
**Pull request by randi274**
# Critical Changes

# Changes
- Progress Indicator
- Step 1: Template Overview
- Step 2: Select Fields
# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by randi274** [comment]
##lurch: attach W-026943

**Comment by randi274** [comment]
##lurch: attach W-026901

**Comment by randi274** [comment]
##lurch: attach W-026900

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -2404,11 +2404,11 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>Time:</value>
     </labels>
     <labels>
-        <fullName>bgeBatchTemplateActiveFields</fullName>
+        <fullName>bgeBatchTemplateSelectedFields</fullName>
```
Comment (bethbrains) [inline src/labels/CustomLabels.labels]: @smarovich bunch of new labels here to look at! thanks!

**Comment by dgamboni** [comment]
Note to self -- Before merging, must test 3 tickets to A/Cs:
W-026900 [PASS]
W-026901 [PASS]
W-026943 [PASS]

**Pull request merged**